### PR TITLE
update arm template to make workspace sku configurable

### LINKED ISSUE #284: Cannot configure workspace sku through pipeline
There's currently no way to configure workspace sku when running the iac pipeline.

### PR DESCRIPTION
**Pull request by mydiemho**
fixes #284

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -58,6 +57,7 @@ The variable group should contain the following required variables. **Azure reso
 | AZURE_RM_SVC_CONNECTION  | azure-resource-connection | [Azure Resource Manager Service Connection](#create-an-azure-devops-service-connection-for-the-azure-resource-manager) name |
 | WORKSPACE_SVC_CONNECTION | aml-workspace-connection  | [Azure ML Workspace Service Connection](#create-an-azure-devops-azure-ml-workspace-service-connection) name                 |
 | ACI_DEPLOYMENT_NAME      | mlops-aci                 | [Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) name                           |
+| WORKSPACE_SKU            | enterprise                | [Workspace Basic vs Enterprise](https://docs.microsoft.com/en-us/azure/machine-learning/overview-what-is-azure-ml#sku)      |
```
[the remark this answers is not in the document]
Comment (dtzar) [reply docs/getting_started.md:60]: Agree, we should minimize the required variables and default to basic/free.  Overall, I think it's good people have the option to deploy enterprise version.

[the remark this answers is not in the document]
Comment (mydiemho) [reply docs/getting_started.md]: addressed your comment

File: docs/getting_started.md [hunk 2]
```diff
@@ -65,6 +64,7 @@ The variable group should contain the following required variables. **Azure reso
 | AZURE_RM_SVC_CONNECTION  | azure-resource-connection | [Azure Resource Manager Service Connection](#create-an-azure-devops-service-connection-for-the-azure-resource-manager) name |
 | WORKSPACE_SVC_CONNECTION | aml-workspace-connection  | [Azure ML Workspace Service Connection](#create-an-azure-devops-azure-ml-workspace-service-connection) name                 |
 | ACI_DEPLOYMENT_NAME      | mlops-aci                 | [Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) name                           |
+| WORKSPACE_SKU            | basic                     | [Workspace Basic vs Enterprise](https://docs.microsoft.com/en-us/azure/machine-learning/overview-what-is-azure-ml#sku)      |
```
Comment (dtzar) [inline docs/getting_started.md:67]: We should remove this from here as we only want required variables here.  This should be added to the .env.example and the variables.yml file.

File: environment_setup/iac-create-environment-pipeline-arm.yml [hunk 3]
```diff
@@ -2,36 +2,35 @@
...
 
 pool:
-  vmImage: 'ubuntu-latest'
+  vmImage: "ubuntu-latest"
 
 variables:
-- group: devopsforai-aml-vg
-
+  - group: devopsforai-aml-vg
+  - name: WORKSPACE_SKU # https://docs.microsoft.com/en-us/azure/machine-learning/overview-what-is-azure-ml#sku
```
Comment (dtzar) [inline environment_setup/iac-create-environment-pipeline-arm.yml:22]: You should put this in the `devopsforai-aml-vg` variable group template which is already included to make it cleaner / all variables in one file.

Comment (mydiemho) [reply environment_setup/iac-create-environment-pipeline-arm.yml:22]: agree that it shouldn't be in the variable group, but don't think it should be in the template either.  The SKU is not use anywhere except in the IAC pipeline which is only rarely run.  

I have put it as a variable in the pipeline.

**Pull request merged**